contracts/interfaces/IAssessment.sol

```diff
@@ -3,180 +3,122 @@
-    uint80 assessmentId,
-    bool accepted,
-    uint32 timestamp,
-    uint96 stakedAmount
-  );
+  function getGroupAssessorCount(uint groupId) external view returns (uint assessorCount);
 
-  function stakeOf(address user) external view returns (
-    uint96 amount,
-    uint104 rewardsWithdrawableFromIndex,
-    uint16 fraudCount
-  );
+  function getGroupAssessors(uint groupId) external view returns (uint[] memory assessorMemberIds);
 
-  function hasAlreadyVotedOn(address voter, uint assessmentId) external view returns (bool);
+  function isAssessorInGroup(uint assessorMemberId, uint groupId) external view returns (bool);
 
+  function getGroupsForAssessor(uint assessorMemberId) external view returns (uint[] memory groupIds);
 
-  /* === MUTATIVE FUNCTIONS ==== */
+  function getGroupsData(uint[] calldata groupIds) external view returns (AssessmentGroupView[] memory groups);
 
-  function stake(uint96 amount) external;
+  function minVotingPeriod() external pure returns (uint);
 
-  function unstake(uint96 amount, address to) external;
+  function payoutCooldown(uint productTypeId) external view returns (uint);
 
-  function unstakeAllFor(address staker) external;
+  function assessorGroupOf(uint claimId) external view returns (uint32);
 
-  function withdrawRewards(
-    address user,
-    uint104 batchSize
-  ) external returns (uint withdrawn, uint withdrawnUntilIndex);
+  function getAssessmentInfo(uint claimId) external view returns (uint acceptVotes, uint denyVotes, uint groupSize, uint32 start, uint32 end, uint32 finalizedAt);
 
-  function withdrawRewardsTo(
-    address destination,
-    uint104 batchSize
-  ) external returns (uint withdrawn, uint withdrawnUntilIndex);
+  function isReadyToCloseAfterVote(uint claimId, Vote vote) external view returns (bool);
 
-  function startAssessment(uint totalReward, uint assessmentDeposit) external returns (uint);
+  function ballotOf(uint claimId, address assessor) external view returns (Ballot memory);
 
-  function castVotes(
-    uint[] calldata assessmentIds,
-    bool[] calldata votes,
-    string[] calldata ipfsAssessmentDataHashes,
-    uint96 stakeIncrease
-  ) external;
+  /* ========= EVENTS ========== */
 
-  function submitFraud(bytes32 root) external;
+  event SetAssessmentDataForProductTypes(uint[] productTypeIds, uint cooldownPeriod, uint groupId);
+  event AddAssessorToGroup(uint indexed groupId, uint assessorMemberId);
+  event RemoveAssessorFromGroup(uint indexed groupId, uint assessorMemberId);  
+  event SetGroupMetadata(uint indexed groupId, bytes32 ipfsMetadata);
 
-  function processFraud(
-    uint256 rootIndex,
-    bytes32[] calldata proof,
-    address assessor,
-    uint256 lastFraudulentVoteIndex,
-    uint96 burnAmount,
-    uint16 fraudCount,
-    uint256 voteBatchSize
-  ) external;
+  event AssessmentStarted(
+    uint indexed claimId,
+    uint32 assessorGroupId,
+    uint32 start,
+    uint32 end
```

Comment:
no point in using uint32 for events, just use full uints, the cast will be done automatically